Src/PCompiler/CompilerCore/Compiler.cs

```diff
@@ -31,6 +31,7 @@ public void Compile(ICompilationJob job)
             Scope scope = Analyzer.AnalyzeCompilationUnit(job.Handler, trees);
 
             // Convert functions to lowered SSA form with explicit cloning
+            IRTransformer.SetOutputLanguage(job.OutputLanguage);
```

Comment:
I dont think we need this anymore. Can we drop this line as we dont do language specific transformations.

Reply:
Also, drop the `SetOutputLanguage` function.

Reply:
(Already) resolved in 1b33a4628d38b61cc2deb68ec6314925fa72855b